tests/rewards/test_reward_nets.py

```diff
@@ -323,20 +359,13 @@ def test_potential_net_2d_obs():
 
 
 @pytest.mark.parametrize("env_name", ENVS)
-@pytest.mark.parametrize("net_cls", REWARD_NETS)
 @pytest.mark.parametrize("num_members", [1, 2, 4])
-def test_reward_ensemble_creation(env_name, net_cls, num_members):
-    """A test RewardEnsemble constructor."""
+def test_reward_ensemble_creation(env_name, num_members):
+    """A simple test of the RewardEnsemble constructor."""
     env = gym.make(env_name)
-    ensemble = reward_nets.RewardEnsemble(
-        env.action_space,
-        env.observation_space,
-        num_members,
-        net_cls,
-    )
+    ensemble = make_ensemble(env.observation_space, env.action_space, num_members)
     assert ensemble
```

Comment:
What is this testing? (That it's not None/False?)

Reply:
This test is extraneous now I've removed it.